apps/fz_http/lib/fz_http_web/controllers/user_controller.ex

```diff
@@ -10,6 +10,12 @@ defmodule FzHttpWeb.UserController do
   def delete(conn, _params) do
     user = Authentication.get_current_user(conn)
 
+    with %{role: :admin} <- user do
+      unless length(Users.list_admins()) > 1 do
+        raise "Cannot delete one last admin"
+      end
+    end
```

Comment:
This is a fail safe. In case 2 admins opened the account page, one deleted their account, and the delete button would still be enabled on the other admin's account page.